components/datetime/src/pattern/runtime/plural.rs

```diff
@@ -201,10 +201,13 @@ impl<'data> PatternPlurals<'data> {
         }
     }
 
+    /// # Panics
```

Comment:
I still am uncomfortable with this. You're hiding the panic away in a helper function. Better would be to expand this function into all of the call sites. But, I am okay if you leave this as a TODO pointing to a new issue (could be the same as the other one for skeletons).

Reply:
I disagree, but we can discuss in the newly filed issue.

Reply:
Filed #2627